### DISCUSSION TITLE
[Incorrect Behavior] layout.arc file for the online arena create menu will not load changes

### DISCUSSION THREAD

**Discussion by kalomaze** [Raytwo/ARCropolis#318] [Issues] [answered]
[ui.zip](https://github.com/Raytwo/ARCropolis/files/8526331/ui.zip)
A texture file was edited on the left side. Reopening in toolbox confirms that it was changed, and the file size definitely changed. Replacing it with a dummy file that's only a few kb large causes a crash on the menu, so it's in the right path and to some degree being patched. Deleting the contents of the main bflyt file in the arc also causes no changes despite the file size being reduced to half as a consequence.

Edit: Solved on my own, read the comment

**Comment by blu-dev** [comment, marked as answer]
I'm not sure if this file is regional, but can you try to give it a regional indicator?

**Reply by kalomaze** [reply]
It doesn't show up as green in CrossARC so I assumed it wasn't regional, and extracting from japanese and english in CrossARC results in identical files according to HxD which confirms this. I added +us_en and got the same results

**Reply by kalomaze** [reply]
Oh my god what it uses the same texture in two different layout.arcs this file format is annoying lmao, nevermind sorry
